Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1645,8 +1645,6 @@ def prepare_PAT(self, sequence = "miniAOD"):
         self.prepare_PATFILTER(self)
         self.loadDefaultOrSpecifiedCFF(sequence,self.PATDefaultCFF)
         self.labelsToAssociate.append('patTask')
-        if not self._options.runUnscheduled:
-            raise Exception("MiniAOD production can only run in unscheduled mode, please run cmsDriver with --runUnscheduled")
```

Comment:
I'd be curious to learn more details on why MiniAOD requires "unscheduled mode" (to understand if not moving all EDProducers to Tasks could cause any problems).

Reply:
PAT was moved to unscheduled-only support a long time ago (was it 2013 or so), hence this check.

Reply:
> PAT was moved to unscheduled-only support a long time ago (was it 2013 or so), hence this check.

Something along that was my recollection as well. Do I understand correctly that it means that as long as PAT is run unscheduled, all is fine (i.e. PAT does not require anything else in the job to be run unscheduled)?

Reply:
> PAT does not require anything else in the job to be run unscheduled)?

yes